test/functional/example_test.py

```diff
@@ -193,10 +193,14 @@ def run_test(self):
 
         self.log.info("Wait for node2 reach current tip. Test that it has propagated all the blocks to us")
 
-        getdata_request = msg_getdata()
-        for block in blocks:
-            getdata_request.inv.append(CInv(2, block))
-        self.nodes[2].p2p.send_message(getdata_request)
+        # If we have Linux perf-tools installed, we can easily profile part of the
```

Comment:
I think this example code should be in `test/README/md`, which is documentation of how to run tests and contains debugging/troubleshooting tips.

This is meant as an example of how to write a test. I don't think the expectation is that people leave `profile_with_perf` contexts in test that are checked into the repo.

Reply:
The presence of usage here is required to get around vulture's dead code check, but I'll add documentation to `test/functional/README.md`.

Reply:
If this is just to get around vulture, then I'd prefer to just update `lint-python-dead-code.sh` to ignore the `profile_with_perf` name. This file is supposed to be a guide for writing a new testcase. As far as I can tell, `profile_with_perf` shouldn't be included in new testcases to be checked into master.